docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -96,14 +96,14 @@ The version numbers is to be set in these Perl modules in the *develop branch*:
 The GUI has no Perl. Update the following files in the *develop branch*:
 
  * zonemaster-gui
-   - [Installation.md][Installation.md GUI]: The version is part of the download
-   path (a directory). It can be repeated several times, once per OS.
-   - [package.json][package.json GUI]: In the top of the file, the version is
-   given after "version".
-   - [environment.prod.ts][environment.prod.ts GUI] and 
-   [environment.ts][environment.ts GUI]: Under "client-info", "version"
-   should point at the version number in both files.
-
+   - [docs/Installation.md][Installation.md GUI]
+     - The version is part of the download path (a directory). It can be
+       repeated several times, once per OS.
+   - [package.json][package.json GUI]
+     - In the top of the file, the version is given after "version".
+     - The file `package-lock.json` is ignored
```

Comment:
Maybe add a final period for consistency? And maybe put `package-lock.json` at the same level as the other files?

Reply:
We can look at that later.